db/migrate/20221019133009_create_likes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] I was unable to set up your app locally because this migration already exists. Kindly remove this file to fix that.